src/components/NFTCover.vue

```diff
@@ -76,6 +91,8 @@ export default {
             'pointer-events-none': !isLoaded,
             'absolute inset-x-0 top-0 opacity-0':
               !isLoaded && !this.isShowVideo,
+            'h-full min-h-[290px] w-full max-w-[200px] object-cover': this
+              .isNftBook,
```

Comment:
This line can be omitted because it inherited the `class` in `imgPropsForPlaceholder` at line 89